package.json

```diff
@@ -54,33 +81,42 @@
     "package-lock.json": "npm install --prefer-offline --no-audit"
   },
   "scripts": {
+    "check-policies": "TESTING=1 binzx/otomi check-policies",
+    "compile": "tsc && chmod +x ./dist/otomi.js",
```

Comment:
Necessary, no. Useful yes, since on first `tsc` doesn't make `./dist/otomi.js` executable.
And chmod luckily isn't expensive.
Do you need it without?